build/ci/job-template.yml

```diff
@@ -191,5 +190,5 @@ jobs:
         artifactName: ${{ parameters.name }} $(_config_short)
         artifactType: container
     - ${{ if eq(parameters.nightlyBuild, 'false') }}:
-      - script: ${{ parameters.buildScript }} -pack -ci
+      - script: ${{ parameters.buildScript }} /p:Build=false -pack -ci  /p:RestorePackagesPath=$(Build.SourcesDirectory)\packages /p:NUGET_PACKAGES=$(Build.SourcesDirectory)\packages
```

Comment:
>/p:Build=false [](start = 46, length = 14)

so with that command we will skip build? If so maybe we should may consider remove -build or even -restore from build script and add -build when we do want to build everything out?

Reply:
Harish and I were just talking about that. we would then have to remember locally to add those in. I think its easier to have it hardcoded in the CI process, then to have every developer remember to do it. 

Do you ahve any suggestions on how to do it so the developer doesn't have to add it? We could create a new build script (maybe ci-build.cmd) that has nothing specified. What would you think of that? @harishsk same question.

Reply:
Hmm, in that case I would prefer keep the build script has nothing specified and add any -option as we need although I don't have strong preference on that.

---
In reply to: [526429534](https://github.com/dotnet/machinelearning/pull/5496#discussion_r526429534) [](ancestors = 526429534)

Reply:
Yes, I'm ok with that one. Please do update the MicrosoftMLOnnxRuntimePackageVersion in eng/Versions.props as I commented in other comments.

---
In reply to: [526481968](https://github.com/dotnet/machinelearning/pull/5496#discussion_r526481968) [](ancestors = 526481968)

Reply:
It seems michael has rebased the arcade branch with the master branch directly some hours ago? Since now the 1.5.2 for onnx appears on the git history of arcade branch:
https://github.com/dotnet/machinelearning/commits/arcade

These changes also seem to have caused some merge conflicts on your other PR as well as a much bigger diff than the one found earlier today on your PR:
https://github.com/dotnet/machinelearning/pull/5494

I'm unsure if this is intended?